vars/rpmOpenSearchDistValidation.groovy

```diff
@@ -46,7 +46,7 @@ def call(Map args = [:]) {
     //Install OpenSearch with designated version via yum
     println("Start installation with yum.")
     rpmCommands(
-            command: "install",
+            command: "sudo env OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123! install",
```

Comment:
Are you expecting user to first define the env var before running `yum/rpm install`?

Reply:
You need to change the spec files of both deb and rpm, check if the var is set else exit with error msgs in the console.

Reply:
Addressed in build repo companion PR.